crates/cext/cbindgen.toml

```diff
@@ -29,3 +34,4 @@ renaming_overrides_prefixing = false
 [export.rename]
 "SparseObservable" = "Obs"
 "CSparseTerm" = "ObsTerm"
+"PyObject" = "PyObject"
```

Comment:
I think `PyObject` is misnamed now, with `renaming_overrides_prefixing` back to `false`.  It would be nice if there was a cleaner way to ask `cbindgen` to just treat a type as "already defined", rather than us needing to hack it in.

Reply:
Hm yes I'm not sure this is possible with `cbindgen`... but given that `PyObject` is available through `Python.h`, which we include if the Python binding is enabled, couldn't we just add `PyObject` to the exclude list? I.e. add
```toml
[export]
exclude = ["PyObject"]
```
and remove it from the renaming section

Reply:
Julien: if that works, that's what I was looking for.

Reply:
Oh fair

Reply:
Done in https://github.com/Qiskit/qiskit/pull/13934/commits/b92f156d9293655df2757ff11e9c8696da544185 per @Cryoris's suggestion

Reply:
Hm crap that doesn't seem to have worked as expected I'm still getting `QkPyObject` used in the header, it's just no longer defined

Reply:
The manual, annoying solution would be to manually do the prefix by renaming all structs we introduce...